### ISSUE TITLE
ProductCrossSellingDefinition: TranslatedField name has required flag - which should only be set on the translation

### ISSUE THREAD

**Issue by AndreasA** [shopware/shopware#975]
**Description**  
The ProductCrossSellingDefinition has the required flag set on the TranslatedField name.

This result in an issue when using upsert in combination with translations that the name for a translation is missing although it was provided.

**Possible Solution**  
Remove the flag from the ProductCrossSellingDefinition and only add it to ProductCrossSellingTranslationDefinition.

**Comment by mitelg** [comment]
hey @AndreasA 
thanks for the report. you are absolutely right. 

Also sounds like an easy pick for a PR contribution :blush: :+1:

**Comment by mitelg** [comment]
@ssltg there are several translatedFields marked as required. this should be fixed and a test for preventing this, would be nice :+1:

**Comment by AndreasA** [comment]
@mitelg  
Just wondering for which branch should the PR be created? 6.2 or master?

Regarding. check. What about adding it to the `dal:validate` command. 
This command could be executed before starting with the tests that would also verify that everything is fine regarding the database (e.g. no migration is missing)

**Comment by mitelg** [comment]
I'm not sure, but I think this could be considered as bug, so a PR against 6.2 should be ok. 

yes, adding this check to the validate command sounds like a good idea :+1:

**Comment by OliverSkroblin** [comment]
The fix can easily be made in a patch release. The existing behavior is only corrected and not changed much.

**Comment by AndreasA** [comment]
Not sure when I will have time to create a PR - even though it is a simple one. Have to do other stuff currently, so it would be nice if someone else could fix this

**Comment by AndreasA** [comment]
I will probably find the time for the PR this weekend.

**Comment by MarcelSchmaeing** [comment]
Hello,

Within our company roadmap and work capacity, we try to address each bug or improvement request but admit that not each one will be resolved. To continue our culture of honesty and openness, we are closing this issue to focus on our roadmap on behalf of all Shopware users.

If you feel like this issue is still impacting you, please create a new issue and let us know.
Thank you again for your valuable feedback.
